src/classes/issues-processor.ts

```diff
@@ -178,7 +187,9 @@ export class IssuesProcessor {
   async processIssue(
     issue: Issue,
     labelsToAddWhenUnstale: Readonly<string>[],
-    labelsToRemoveWhenUnstale: Readonly<string>[]
+    labelsToRemoveWhenUnstale: Readonly<string>[],
+    labelsToAddWhenStale: Readonly<string>[],
+    labelsToRemoveWhenStale: Readonly<string>[]
```

Comment:
The `processIssue` function processes all stale issue thus it will try to remove labels for all stale issues. I think we need to remove labels once when we mark issue as a stale in the `_markStale` function

Reply:
discussed